pkg/infra/usagestats/service/accesscontrol.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+const (
+	ActionRead = "server.stats.report:read"
```

Comment:
Not sure what's a good and clear name for this action. Is it `server.stats` or `server.usagestats`?

Reply:
`usagestats` would be most correct as there's already server stats which is a different endpoint